docs/src/learning_networks.md

```diff
@@ -748,20 +748,19 @@ Q = @node sqrt(Z)
 (so that `Q() == 4`). Here's a more complicated application of `@node` to row-shuffle a
 table:
 
-```julia
-using Random
-X = (x1 = [1, 2, 3, 4, 5],
-         x2 = [:one, :two, :three, :four, :five])
-rows(X) = 1:nrows(X)
+```julia-repl
+julia> using Random
+julia> X = (x1 = [1, 2, 3, 4, 5],
+            x2 = [:one, :two, :three, :four, :five])
+julia> rows(X) = 1:nrows(X)
 
-Xs = source(X)
-rs  = @node rows(Xs)
-W = @node selectrows(Xs, @node shuffle(rs))
+julia> Xs = source(X)
+julia> rs = @node rows(Xs)
+julia> W = @node selectrows(Xs, @node shuffle(rs))
 
```

Comment:
Generally, the pattern has been to only include  `julia> ` before input when corresponding output is to follow. Are you finding this confusing?

Reply:
Good to have this feedback, thank you. 

Unfortunately, this convention is pretty widespread, at least in the MLJ ecosystem. I wonder if others are also finding this confusing. @OkonSamuel @EssamWisam What's you view of this? Do we need to include `julia>` prompts in every single line when there is sometimes REPL output to be included in a docstring? Often we just only add `julia>` before lines with output immediately following.

Reply:
My preference is to minimize the use of `>julia` as it jams up the code (and has to be dropped before copy-pasting into `.jl` files) which as you said is consistent with the ongoing convention for `MLJ`. That said, I do find @abhro to be valid; it would be not immediately obvious to a beginner that the printed object is not an actual function being called and for that I think we could expose to readers the convention that what immediately comes after a line with `>julia` is the output of the code in that line.

In any case, in the future, we can explore the possibility of having the outputs be printed in a differently styled cell as in [Imbalance tutorials](https://juliaai.github.io/Imbalance.jl/dev/examples/smoten_mushroom/smoten_mushroom/) and [DataScienceTutorials](https://juliaai.github.io/DataScienceTutorials.jl/getting-started/model-tuning/) which would allow dropping all `>julia` and would be optimal for maximal facilitation of code reuse.

Reply:
Yes, I think the way the `@example` macro does it also provides a good separation between output and code.